src/detect-bytetest.c

```diff
@@ -1279,6 +1369,59 @@ static int DetectBytetestTestParse22(void)
     return result;
 }
 
+/**
+ * \test Test bitmask option.
+ */
+static int DetectBytetestTestParse23(void)
+{
+    int result;
+    DetectBytetestData *data = NULL;
+    data = DetectBytetestParse("4, <, 5, 0, bitmask 0xf8", NULL, NULL);
+    result = data
+             && data->op == DETECT_BYTETEST_OP_LT
+             && data->nbytes == 4
+             && data->value == 5
+             && data->offset == 0
+             && data->flags & DETECT_BYTETEST_BITMASK
+             && data->bitmask == 0xf8
+             && data->bitmask_shift_count == 3;
+    if (data)
```

Comment:
can be conditionless if we would have `FAIL_IF_NULL(data)` above

Reply:
Will update.